Cargo.toml

```diff
@@ -17,7 +17,13 @@ download_ffmpeg = ["reqwest"]
 [lib]
 crate-type = ["lib"]
 
+[target.'cfg(target_os = "linux")'.dependencies]
+tar = "0.4.42"
```

Comment:
Oh yeah, one other small thing. What do you think about making these 3 deps optional, and grouping them all underneath the `download_ffmpeg` feature?

My idea is that if someone has opted out of the download functionality, maybe they have their own plans about how to acquire and extract the binary and they don't need these dependencies. So the `download_ffmpeg` feature could look like: `download_ffmpeg = ["reqwest", "zip", "tar", "xz2"]`. Do you agree?

Reply:
Affirmative, makes more sense actually. Let me make this change